jbmc/regression/jbmc-strings/string-non-empty-option/test_non_empty.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 Test.class
---function Test.checkMinLength --string-non-empty
+--function Test.checkMinLength --string-non-empty --max-nondet-string-length 1000000000
```

Comment:
This is not big enough to cause an overflow (which is what one of the assertion is checking here), the maximum java integer is 2147483647 so you should use at least half that, 2000000000 should work

Reply:
I agree with @romainbrenguier about the overflow.  However, if this is about finding overflows / checking if they occur, please could it say that somewhere in the test or the commit message.

Reply:
Thank you, I adjusted the value

Reply:
The java file already say:
```
 // For this assertion to be valid, also need to limit the length of
 // strings because overflows could cause the sum to be negative.
 assert arg1.length() + arg2.length() > 1;
```
maybe the desc file should mention that we explicitly set the length limit high to be able to detect overflow behaviors.

Reply:
I added explanation to the `.desc` files about the overflow